components/centraldashboard/app/prometheus_metrics_service.ts

```diff
@@ -0,0 +1,90 @@
+import {Interval, MetricsInfo, MetricsService, TimeSeriesPoint} from "./metrics_service";
+import {PrometheusDriver, RangeVector, ResponseType} from 'prometheus-query';
+
+export class PrometheusMetricsService implements MetricsService {
```

Comment:
This seems like an interesting service that they are introducing. 
Did you see any of this being used in your investigation of kf1.9?

Reply:
The new feature is this https://github.com/kubeflow/kubeflow/pull/7116. It seems that by default it is 'undefined' since we will need to add urls and things. I am unsure if we want it or not. I figured that we could simply remove the app if we did not.

Reply:
Issue created https://jirab.statcan.ca/browse/BTIS-701